apps/web/src/config/constants/exchange.ts

```diff
@@ -4,27 +4,29 @@ import { bitgertTokens, dogechainTokens, dokenTokens, fuseTokens, xdcTokens } fr
 import { ChainMap, ChainTokenList, RouterAddressTypes } from './types'
 
 export const ROUTER_ADDRESS_COMMON = '0xBb5e1777A331ED93E07cF043363e48d320eb96c4'
-export const ROUTER_ADDRESS_COMMON_AKKA = '0x9B13efdA353D1f2B127270A198e2960841e899B0'
+export const ROUTER_ADDRESS_COMMON_AKKA_BITGERT = '0x9B13efdA353D1f2B127270A198e2960841e899B0'
+export const ROUTER_ADDRESS_COMMON_AKKA_XDC = '0xB8727B821e17FC125CC4656BaD7181EF84a3dB7e'
+
 export const ROUTER_ADDRESS: ChainMap<RouterAddressTypes> = {
   [ChainId.BITGERT]: {
     Icecream: ROUTER_ADDRESS_COMMON,
-    Akka: ROUTER_ADDRESS_COMMON_AKKA,
+    Akka: ROUTER_ADDRESS_COMMON_AKKA_BITGERT,
   },
   [ChainId.DOGE]: {
     Icecream: ROUTER_ADDRESS_COMMON,
-    Akka: '',
+    Akka: ROUTER_ADDRESS_COMMON,
   },
   [ChainId.DOKEN]: {
     Icecream: ROUTER_ADDRESS_COMMON,
-    Akka: '',
+    Akka: ROUTER_ADDRESS_COMMON,
```

Comment:
Just don't set router address for AKKA if there is none instead of defaulting to the normal router, as they have different interfaces, using our router with the AKKA code would result in weird crashes that would be very hard to debug.

Reply:
Please make sure that the empty string does not cause problems for your app.

Reply:
the empty string certainly will cause problems for the app, that's why I requested to completely remove the AKKA entry for unsupported chains. The AKKA router address should not be loaded on unsupported chains and if it is, it should crash immediately instead of cause weird issues due to wrong addresses or empty string addresses.

Reply:
What is your solution to fix this problem? Can you explain more?

Reply:
I want to know if you have any idea for this?

Reply:
The suggestion is to not define AKKA router address for chains where it does not exist. If it gets accessed somewhere, it should Intentionally crash due to that address should only be read for supported chains. If you read an empty string as address, it won't crash immediately but further down the code, where it will be hard to find the actual issue

Reply:
So dont define AKKA for chains that don't have AKKA support.

Reply:
I made the type of AKKA section optional in the ROUTER ADDRESS and removed it from chain objects that cannot be supported for now.
Of course, there was another way to consider typing undefined for AKKA.
Whatever you prefer, we will act